patches/series

```diff
@@ -16,6 +16,5 @@ post-install.diff
 log-level.diff
 local-storage.diff
 service-worker.diff
-last-opened.diff
```

Comment:
Funny enough, they removed the same last-opened logic that we did which means we were able to remove our patch!

https://github.com/microsoft/vscode/commit/6124b1016661c0cc3eb79f04a7ffb7d7dd2a1439

https://github.com/microsoft/vscode/pull/146364